library/Recurly/V3/Webhook/Webhook.hs

```diff
@@ -134,6 +138,58 @@ documentToNotification document = Foldable.asum $ fmap
   , getDunningInfo DunningEventNew "new_dunning_event_notification"
   ]
 
+extractAccountCode :: Notification -> Account.Code
+extractAccountCode notification = case notification of
```

Comment:
I really don't like this but I could not figure out another way for the design as it stands

Reply:
It's a bit beefy but I don't have a suggestion currently, for our purposes I think this should be fine.

Reply:
There's no other way to write this. Looks reasonable to me, except that you don't need parens around the pattern matches and you could use a shorter variable name (like `x`) to make it visually simpler.